convert2rhel/main.py

```diff
@@ -92,7 +92,7 @@ def main():
         # restart system if required
         utils.restart_system()
 
-    except (Exception, SystemExit, KeyboardInterrupt), err:
+    except (Exception, SystemExit, KeyboardInterrupt) as err:
```

Comment:
Using "," is necessary for python 2.4 compatibility (RHEL 5). I believe we don't need the code to be Py3-compatible as of yet. Maybe later if we go for conversions to RHEL 8.

Reply:
Returned it back as it was